docs/release-notes/release-notes-0.17.0.md

```diff
@@ -10,3 +10,10 @@
 
 * Jordi Montes
 
+## Neutrino
+
+* Added [`unbanPeer`](https://github.com/lightningnetwork/lnd/pull/7606) command to lncli
+
+# Contributors (Alphabetical Order)
+
+* Ononiwu Maureen Chiamaka
```

Comment:
your name should go into the existing `# Contributors` section. And the `# Neutrino` section should go above the contributors section. See previous release note docs for examples :)

Reply:
Okay thank you